go/libkb/usercard_cache.go

```diff
@@ -0,0 +1,62 @@
+package libkb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+
+	"stathat.com/c/ramcache"
+)
+
+// UserCardCache caches keybase1.UserCard objects in memory.
+type UserCardCache struct {
```

Comment:
if we're going to cache sig hints too, maybe can the cache be more generic, and cache at the HTTP-level?